_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP and why is it so important to do a pre-GSoC analysis
+and design it as early as possible, for it to be finalized in the cEP?"
```

Comment:
I am having trouble understanding the sentence. Doesn't the "it" in `design it` refer to the cEP itself?

Reply:
yes, it is being referred to cEP itself.

Reply:
And what does the "it" in `for it to be finalized in the cEP?` refer to?

Reply:
Changing `it` to `a proposal` for better understanding. Committing with some new changes,please  check it and review, if it is still not understandable.

Reply:
"it" in for it to be finalized in the cEP , is also referring to "cEP"